src/cleanlab_codex/internal/validator.py

```diff
@@ -108,3 +132,38 @@ def is_bad(metric: str) -> bool:
     if is_bad("trustworthiness"):
         return "hallucination"
     return "other_issues"
+
+
+def validate_messages(messages: Optional[list[dict[str, Any]]] = None) -> None:
```

Comment:
I think this name `validate_messages` should be more carefully chosed when the entire module reserves the name method `validate` in Validator to just look at the trust & Eval scores.

I'd bed we wouldn't change the Validator.validate api, but we could find a different name for `validate_messages` since it behaves quite differently.